src/mimeparser.rs

```diff
@@ -213,6 +213,9 @@ pub enum SystemMessage {
 
     /// This message contains a users iroh node address.
     IrohNodeAddr = 40,
+
+    /// "Messages are end-to-end encrypted."
+    ChatE2ee = 50,
```

Comment:
Question (non-blocking): This is the same string as `ChatProtectionDisabled`; what's the plan for these to strings?

Is it in case we want to have a different string for `ChatProtectionEnabled` in the future? Or it will stay the same, and finally replace `ChatProtectionEnabled`? In this case, why not deprecate `ChatProtectionEnabled`, and stop using it? Or would you like to do this later?

Reply:
this is not the string, but the info-message-type - eben if both have the same string, ui may want to show a green checkmark or not (at least currently)